assets/js/sections/logbookadvanced.js

```diff
@@ -27,6 +27,25 @@ $('#selectPropagation').change(function () {
 	}
 });
 
+//Converts UTC time to local time.
+//This accounts for Daylight Savings (apparently)
+function UTCtoLocal(datetime, timezone) {
+    const [datePart, timePart] = datetime.split(' ');
+    const [day, month, year] = datePart.split('/').map(Number);
+    const [hour, minute] = timePart.split(':').map(Number);
+    const fullYear = year < 100 ? 2000 + year : year;
+    const utcDate = new Date(Date.UTC(fullYear, month - 1, day, hour, minute));
+    const localDate = new Date(utcDate.toLocaleString('en-US', { timeZone: timezone }));
+    const formattedDate = [
+        String(localDate.getDate()).padStart(2, '0'),  // Day
+        String(localDate.getMonth() + 1).padStart(2, '0'),  // Month
+        String(localDate.getFullYear()).slice(-2),  // Year
+    ].join('/') + ' ' + [
+        String(localDate.getHours()).padStart(2, '0'),  // Hours
+        String(localDate.getMinutes()).padStart(2, '0'),  // Minutes
+    ].join(':');
+    return formattedDate;
+}
```

Comment:
You decided to write a second function which converts UTCtoLocal in Javascript. 
In this case this is not necessary as the data is coming from the controller. Why not converting in the controller where the PHP function is available.